Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+  - arguments:
+    - default: false
+      isArray: false
+      name: id
+      required: true
+      description: The custom URL category id.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Delete a specific url category.
+    name: prisma-sase-custom-url-category-delete
+  - arguments:
+    - default: false
+      description: A specific external dynamic list to return. If not specified, all external dynamic lists will be returned.
+      isArray: false
+      name: id
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - default: false
+      isArray: false
+      name: limit
+      description: The maximum number of results to return. Default is 50.
+    - default: false
+      isArray: false
+      name: page
+      description: Page number you would like to view. Each page contains page_size values. Must be used along with page_size.
+    - default: false
+      isArray: false
+      name: page_size
+      description: Number of results per page to display.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Lists all external dynamic lists.
+    name: prisma-sase-external-dynamic-list-list
+    outputs:
+    - contextPath: PrismaSase.ExternalDynamicList.id
+      description: The external dynamic list id.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.name
+      description: The external dynamic list name.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.folder
+      description: The external dynamic list folder.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.description
+      description: The external dynamic list description.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.type
+      description: The external dynamic list type.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.source
+      description: The external dynamic list source.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.frequency
+      description: The external dynamic list frequency.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: name
+      required: true
+      description: The external dynamic list name.
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this a folder location or folder type/name?